src/Transactions/Transfer/TransferAddModalForm.js

```diff
@@ -50,17 +93,21 @@ const TransferAddModalForm = ({ handleSubmit, onClose, funds }) => {
         <Row>
           <Col xs>
             <FieldSelect
-              dataOptions={funds}
+              dataOptions={getOptionsFrom()}
               label={<FormattedMessage id="ui-finance.transaction.transferFrom" />}
               name="fromFundId"
+              onChange={selectFromFund}
+              required={isTransferFromReqired}
+              {...validateTransferFrom}
             />
           </Col>
 
           <Col xs>
             <FieldSelect
-              dataOptions={funds}
+              dataOptions={getOptionsTo()}
```

Comment:
please don't do like this. You can define all required options above and pass here variables

Reply:
fixed